scripts/generate-list-of-rules.js

```diff
@@ -0,0 +1,41 @@
+#!/usr/bin/env node
+
+'use strict';
+
+const fs = require('fs');
+const {rules} = require('../index');
+
+const ruleListItems = Object.keys(rules)
+  .sort()
+  .map((id) => {
+    const {meta} = rules[id];
+    return `* [react/${id}](docs/rules/${id}.md): ${meta.docs.description}${
+      meta.fixable ? ' (fixable)' : ''
+    }`;
+  });
+
+const targetFile = process.argv[2];
+const targetFileContent = fs.readFileSync(targetFile, 'utf8');
+
+const NOTICE = '<!-- THIS SECTION IS AUTO-GENERATED. DO NOT EDIT. -->';
+const TAG_BASIC_RULES = '<!-- BASIC_RULES_SECTION -->';
+const TAG_JSX_RULES = '<!-- JSX_RULES_SECTION -->';
+
+const newContent = targetFileContent.replace(
```

Comment:
perhaps it would be worth using something like https://www.npmjs.com/package/markdown-magic rather than messing with regexes?

Reply:
Looks good! I will try it on this PR later, thanks! 👍

Reply:
I will try it with #1742.